plasma_framework/docs/design/ald-exit-game-definitions-and-properties.md

```diff
@@ -0,0 +1,133 @@
+# ALD Exit Game Definitions And Properties
+
+## New definitions from ALD (abstract layer design)
+Based on [MoreVP document](https://github.com/omisego/elixir-omg/blob/master/docs/morevp.md). This section adds some new definitions introduced from ALD or give explicit names to some of the implicit characteristics of the origin MoreVP spec.
+
+### SpendingCondition Fulfilled
+1. Checks an input tx and the spending tx, whether the spending tx fulfil the requirement of spending the output of input tx or not.
+1. Basic checks:
+    - Spending tx input points to the output of input tx.
+    - Spending tx fulfil the “lock” condition of the output
+1. This checks on (at most) tx level, does not check things on Plasma protocol level.
+
+### Tx Standard Finalization
+1. MVP:
+    - Tx is in a plasma block. (Checks inclusion proof of plasma block)
+    - It is confirmed by user. (Checks confirm signature is valid for the inclusion block)
+2. MoreVP:
+    - Tx is in a plasma block. (Checks inclusion proof of plasma block)
+
+### Tx Protocol Finalized
+This is the minimal finalization requirement according to each protocol.
+
+1. MoreVp tx: the transaction exists
+2. MVP tx: the transaction is standard finalized
+
+## Exit Properties
+
+### General Properties
+
+#### Valid Spending Tx:
+1. Needs to be protocol finalized
+1. Fulfils the spending condition of the output
+
+#### Input/Output finalized
+1. The input/output has been withdrawn/processed.
+1. Or The output has been used as an input of a processed IFE.
+
+#### Input/Output withdrawable
+The input/output is not finalized already
+
+### In-flight Exit
+
+#### InFlight exit to be exitable
+1. Inputs of the exiting tx should be Standard Finalized (Protocol finalized is not enough)
+1. The state transition of the tx is valid
+1. All inputs fulfill the spending condition with the exiting tx
+1. The transaction itself is Protocol finalized
+
+#### Competing tx
+1. Needs to be protocol finalized
+1. Fulfils the spending condition of an input of the InFlight exiting tx
+
+#### Canonical tx
+1. The tx is exitable
```

Comment:
Since we have the current work going on with locking down the format of tx. Can we add a detailed specification for every attribute in tx and on what makes it an exitable tx?

Reply:
Add a section of tx in `payment-game-implementation-v1`, can you take a look?